src/components/Dialog/Dialog.component.tsx

```diff
@@ -0,0 +1,6 @@
+import { memo } from "react";
+const Dialog = () => {
+  return <div className="">Dialog is updated</div>;
+};
+
+export const MemoDialog = memo(Dialog);
```

Comment:
@sultondev why didn't you disable an eslint naming rule and export directly the memorized component?

Reply:
it is okay bro, just chill